manual_requirements.txt

```diff
@@ -1,3 +1,3 @@
-    pytket[ZX] == 1.20.1
```

Comment:
I think this is included in some manual file and the spaces before the module names are needed for the include to look right?

Reply:
Okay I changed it back.

Reply:
![Screenshot 2023-10-23 at 17 35 37](https://github.com/CQCL/pytket/assets/93673602/587cbb2e-5966-4a67-b936-66d4bc9dfdb6)

Fixed now.